rest/src/main/java/com/confighub/api/repository/client/v1/APIPush.java

```diff
@@ -345,6 +346,9 @@ else if ( repository.isValueTypeEnabled() && !hasKeyContextDeleteOpp(gson, entry
                                   : null;
 
                     store.deleteKeyAndProperties( appName, repository, token, key, pass, changeComment );
+                    ConcurrentContextPropertiesCache.getInstance().keySet().stream()
```

Comment:
do we only want to do this when repository caching is enabled?

Reply:
If caching is not enabled, it'll only be removing non-existent elements.  I don't think we need a feature flag around that since it doesn't affect business logic.